Gruntfile.js

```diff
@@ -70,6 +70,7 @@ module.exports = function( grunt ) {
 					ajax: [ "manipulation/_evalUrl", "event/ajax" ],
 					callbacks: [ "deferred" ],
 					css: [ "effects", "dimensions", "offset" ],
+					"css/showHide": [ "effects" ],
```

Comment:
If you don't remove effects, showHide will be [pulled back in](https://github.com/jquery/jquery/blob/67b76f5986a78d868553b9aec0d0192f5259a078/src/effects.js#L11) I think?

On the opposite side, you can remove effects but still want to use the non-animated `.show()` and `.hide()` methods so I don't think it should be force-removed.